oauth2_client/tests/test_client.py

```diff
@@ -22,6 +19,10 @@ def test_no_token(self, get_token_mock):
         """
         Ensure new token is created if there is no token.
         """
+        from oauth2_client.models import AccessToken
```

Comment:
Are there benefits of having local imports rather than global?

Reply:
here unfortunately yes. When the test imports django models, django has to be setup before such an import happens. Otherwise an exception is thrown. When running a unit test from IDE, django.setup() runs _after_ test files are imported. So the only way to import model classes after django has setup, was to make imports local. 

However if anyone knows a better way that can avoid local imports, I am happy to change that. @labstersteve  ?

Reply:
This is weird, as far as I know it's not a default behaviour CMIIR. If it's really so, please specify this as a comment.

Reply:
@fredmajor as it is only for IDE case, please leave comment about it.